### ISSUE TITLE
hasPermission() occurs Exception in android

### ISSUE THREAD

**Issue by tree1891** [llfbandit/record#85]
if permission not allowed, hasPermission() occurs Exception.

```
E/flutter ( 7551): [ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: PlatformException(-2, Permission denied, null, null)
E/flutter ( 7551): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:607:7)
E/flutter ( 7551): #1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:167:18)
E/flutter ( 7551): <asynchronous suspension>
E/flutter ( 7551): #2      MethodChannelRecord.hasPermission (package:record_platform_interface/src/method_channel_record.dart:13:20)
E/flutter ( 7551): <asynchronous suspension>
```

**Comment by llfbandit** [comment]
I will remove the exception in next version. Thanks for the report.

**Comment by llfbandit** [comment]
Fixed in record 4.3.2
